### PR TITLE
Doc examples for file_download

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #575.

+ Release Notes restructuration

### CODE REVIEW

**Review by jrobinAV — APPROVED** [review]
I only reviewed rel notes

File: docs/manuals/gui/viselements/file_download.md_template [hunk 1]
```diff
@@ -37,11 +41,13 @@ Allows downloading _content_ when content is a file path or some content.
 
 ### Standard configuration
 
-A specific _label_ can be shown beside the standard icon.
+The [*label*](#p-label) property can be set to specify the label used as the button text next to
+the download icon.
 
-The function name provided as _on_action_ is called when the user initiates the download.
+The function name provided as *on_action* is called when the user initiates the download.
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/file_download.md_template]: not initiates except if content is None ?

File: docs/manuals/gui/viselements/file_download.md_template [hunk 2]
```diff
@@ -59,14 +65,15 @@ The _name_ provided will be the default name proposed to the user when downloadi
...
+The file content can be visualized in the browser (if supported and in another tab) by setting
+*bypass_preview* to False.
 
 !!! example "Page content"
 
     === "Markdown"
 
         ```
-        <|{content}|file_download|bypass_preview=False|>
+        <|{content}|file_download|dont bypass_preview|>
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/file_download.md_template]: dont => don't

**Review by FredLL-Avaiga — APPROVED** [review]
with comments

File: docs/manuals/gui/viselements/file_download.md_template (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/file_download.md_template]: O..k...

File: docs/manuals/gui/viselements/file_download.md_template (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/file_download.md_template]: Correct!

**Pull request merged**